openvm/tests/optimizer.rs

```diff
@@ -55,3 +55,33 @@ fn test_optimize() {
         [3194, 2861, 160]
     );
 }
+
+#[test]
+fn test_conflicting_constraints_in_bus_interaction() {
+    let file =
+        std::fs::File::open("tests/conflicting_constraints_in_bus_interaction.cbor").unwrap();
```

Comment:
hmm do we wanna keep artifacts like these in the repo? Feels a bit odd since we can't read its contents, so the test doesn't mean much

Reply:
A better way would be to add unit tests, but I won't add unit tests for units I have not written...